packages/ifpack2/src/Ifpack2_LocalSparseTriangularSolver_decl.hpp

```diff
@@ -107,6 +107,9 @@ class LocalSparseTriangularSolver :
   //! Specialization of Tpetra::RowMatrix used by this class.
   typedef Tpetra::RowMatrix<scalar_type, local_ordinal_type,
                             global_ordinal_type, node_type> row_matrix_type;
+  //! Specialization of Tpetra::CrsMatrix used by this class.
+  typedef Tpetra::CrsMatrix<scalar_type, local_ordinal_type,
+                            global_ordinal_type, node_type, false> crs_matrix_type;
```

Comment:
You may leave off the fifth template parameter `false`.  We intend to get rid of that at some point, and it has the right default value.